### PR TITLE
VLE/KBE-39/conversion_dtypes

### PR DESCRIPTION
**Pull request by Akaud**
Automatic dtyping for lovd and gnomad without using constants.py

- renamed back-end to back_end
- `pandas.DataFrame.convert_dtypes` is used for automatic data conversion

### CODE REVIEW

File: app/back_end/data/refactoring.py [hunk 1]
```diff
@@ -10,38 +10,53 @@
...
-                case "Integer":
-                    frame[column] = pd.to_numeric(frame[column]).astype('Int64')
-                case "Double":
-                    frame[column] = pd.to_numeric(frame[column]).astype('float')
                 case _:
-                    raise ValueError(f"Undefined data type: "
-                                     f"{LOVD_TABLES_DATA_TYPES[table_name][column]}")
-
+                    raise ValueError(f"Undefined data type for column '{column}': {frame[column].dtype}")
+        df_dict[table_name] = frame
```
Comment (Strexas) [inline app/back_end/data/refactoring.py:59]: You already converted with using `convert_dtypes`. Why do you continue to convert?

File: app/back_end/data/refactoring.py [hunk 2]
```diff
@@ -50,22 +65,22 @@ def set_gnomad_dtypes(df):
...
-                df[column] = pd.to_numeric(df[column], errors='coerce').astype('Int64')
-            case "Double":
-                df[column] = pd.to_numeric(df[column], errors='coerce').astype('float')
+            case _ if pd.api.types.is_datetime64_any_dtype(df[column]):
+                df[column] = pd.to_datetime(df[column], errors='coerce')
+            case pd.BooleanDtype():
+                df[column] = df[column].map({"0": False, "1": True}).astype('boolean')
             case _:
-                raise ValueError(f"Undefined data type: {GNOMAD_TABLES_DATA_TYPES[column]}")
+                raise ValueError(f"Undefined data type for column '{column}': {df[column].dtype}")
```
Comment (Strexas) [inline app/back_end/data/refactoring.py:83]: same

**Review by Strexas — CHANGES_REQUESTED** [review]

File: app/back_end/data/refactoring.py (hunk 1 above)
Comment (Akaud) [reply app/back_end/data/refactoring.py:59]: Because it automatically just converts everything to string

**Comment by mantvydasdeltuva** [comment]
Would appreciate for you to not change root directories names as you did for `back-end` directory. This can cause unexpected problems down the road. @Akaud

**Pull request closed without merging**